wurst/file/ByteBuffer.wurst

```diff
@@ -0,0 +1,145 @@
+package ByteBuffer
+
+import Table
+import Bitwise
+
+/*	ByteBuffer is a utility class that allows you to write/read bytes, shorts and integer to/from it.
+	It may be useful for compacting bytes into integers or decomposing integers into bytes.
+*/
+
+public class ByteBuffer
+	private var storage = new Table()
```

Comment:
why are you using a Table if you only load/store ints?

Reply:
Could have used HashMap, but (I'm a little bit paranoid probably :D) just chose the simplest thing satisfying the needs (since I don't ever remove the elements, don't need to increment/decrement the size, etc.) + tried to avoid some extra overhead the the HashMap offers, i.e. size tracking with `has` checks on the every `put` method. Should not matter a lot, though may when there is over 500k write/read ops (for 500 Kb file for example).

Reply:
Use HashMap